CHANGES/10759.packaging.rst

```diff
@@ -0,0 +1,5 @@
+Support building against system llhttp library -- by :user:`mgorny`
```

Comment:
We could mark this as inline code, which should exclude it from spellchecking..
Also, add a trailing period when the byline is a part of a sentence and doesn't stand alone on a dedicated line.
```suggestion
Added support for building against system ``llhttp`` library -- by :user:`mgorny`.
```

Reply:
Well, sure. I originally followed suit with "aiohttp" being mentioned as plain text.